ChameleonForms/FieldGenerators/Handlers/DateTimeHandler.cs

```diff
@@ -36,7 +37,15 @@ public override IHtmlString GenerateFieldHtml(IReadonlyFieldConfiguration fieldC
         public override void PrepareFieldConfiguration(IFieldConfiguration fieldConfiguration)
         {
             if (!string.IsNullOrEmpty(FieldGenerator.Metadata.DisplayFormatString))
-                fieldConfiguration.Attr("data-val-format", FieldGenerator.Metadata.DisplayFormatString.Replace("{0:", "").Replace("}", ""));
+            {

```

Comment:
Ahhhh right, so the problem is jQuery doesn't support it. What I usually do is use the full date format string rather than use the shorthand ones (in terms of readability/documentation in the view model I think its better too).

One question: why do you want to tie it to the culture of the server it's deployed? Shouldn't it be baked into the app what format you want. What if someone changed the culture on the server on you and then users got confused?